packages/frontend/src/components/FlowSubstep/index.tsx

```diff
@@ -79,13 +96,13 @@ function FlowSubstep(props: FlowSubstepProps): React.ReactElement {
 
   const { name, arguments: args } = substep
 
-  const editorContext = React.useContext(EditorContext)
+  const editorContext = useContext(EditorContext)
   const formContext = useFormContext()
-  const [validationStatus, setValidationStatus] = React.useState<
-    boolean | null
-  >(validateSubstep(substep, formContext.getValues() as IStep))
+  const [validationStatus, setValidationStatus] = useState<boolean | null>(
```

Comment:
actly when would this be null ah? seems like its always a boolean?

Reply:
good catch. It'll never be null. updated